lib/src/splash.dart

```diff
@@ -93,26 +91,30 @@ class _FlameSplashScreenState extends State<FlameSplashScreen> {
       constraints: widget.theme.constraints,
       decoration: widget.theme.backgroundDecoration,
       child: Center(
-          child: ValueListenableBuilder<int>(
-        valueListenable: controller._stepController,
-        builder: (context, currentStep, _) {
-          if (currentStep == null) {
-            return Container();
-          }
-          return _SplashScreenStep(
-            builder: steps[currentStep],
-            durations: controller._durations,
-            key: ObjectKey(currentStep),
-          );
-        },
-      )),
+        child: ValueListenableBuilder<int?>(
+          valueListenable: controller._stepController,
+          builder: (context, currentStep, _) {
+            if (currentStep == null) {
```

Comment:
Can `currentStep` be null?

Reply:
In the previous implementation, while idle, yes. changed to be zero